app/services/article_api_index_service.rb

```diff
@@ -71,6 +75,16 @@ def tag_articles
     articles.page(page).per(per_page || DEFAULT_PER_PAGE)
   end
 
+  def tagged_articles
+    articles = Article.published.includes(:user, :organization)
+    articles = articles.tagged_with(tags) if tags
```

Comment:
I just confirmed that tags do not need to be passed as an array, thanks to:
```
ActsAsTaggableOn.default_parser.new("test1, test2,test3").parse
=> ["test1", "test2", "test3"]
```
https://github.com/mbleigh/acts-as-taggable-on/blob/950c01072b7ab6227d74dc0d0b112f0135281fc2/lib/acts_as_taggable_on/taggable/core.rb#L111